guide/popular-topics/faq.md

```diff
@@ -315,6 +315,23 @@ const user = <message>.mentions.users.first();
 Mentions in embeds may resolve correctly in embed description and field values but will never notify the user. Other areas do not support mentions at all.
 :::
 
+### How do I control which users and/or roles are mentioned in a message?
+
+Controlling which mentions (and replies) will send a ping is done via the `allowedMentions` option, which replaces `disableMentions`.
```

Comment:
```suggestion
Controlling which mentions will send a ping is done via the `allowedMentions` option, which replaces `disableMentions`.
```
Since the `allowedMentions` options works with `channel.send()` but not `message.reply()` in v12, the "(and replies)" bit should be removed for now.